.github/workflows/release.yml

```diff
@@ -12,32 +12,11 @@ env:
   miniforge_variant: "Mambaforge"
 
 jobs:
-  pypi:
-    name: Publish to PyPI
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check out repo
-        uses: actions/checkout@v4
-        with:
-          submodules: true
-
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: ${{ env.earliest_python }}
-
-      - name: Build distribution
-        run: |
-          export RELEASE_VERSION=${{ github.ref_name }}
-          pip install numpy cython
-          python setup.py sdist
-
-      - name: Publish distribution
-        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-        uses: pypa/gh-action-pypi-publish@release/v1
-        with:
-          user: __token__
-          password: ${{ secrets.PYPI_API_TOKEN }}
+  # Trigger wheel building workflow
+  build_and_publish:
+    name: Build and publish to PyPI
+    uses: ./.github/workflows/wheels.yml
```

Comment:
The filename here `wheels.yml` and the actual filename `build_wheels.yml` conflict. (btw, I prefer `wheels.yml`)